### ISSUE TITLE
Uncaught Error: ENOENT: no such file or directory, OPEN  /home/ely/Documents/Coding/Bash Mincraft Installer/Enkouyami s Bash Minecraft Installer 

### ISSUE THREAD

**Issue by Enkouyami** [atom/autosave#33]
Steps to reproduce below: 
Opening a broken symlink of a file

**Atom Version**: 0.194.0
**System**: linux 3.13.0-37-generic
**Thrown From**: autosave https://github.com/atom/autosavepackage, v0.20.0
### Stack Trace Uncaught Error:
```
ENOENT: no such file or directory, OPEN/home/ely/Documents/Coding/Bash Mincraft Installer/Enkouyami s Bash Minecraft Installer
At /opt/atom/resources/app.asar/node_modules/pathwatcher/lib/file.js:376
Error: ENOENT: no such file or directory, OPEN/home/ely/Documents/Coding/Bash Mincraft Installer/Enkouyami s Bash Minecraft Installerat Errornativeat Object.fs.OPENSyncfs.js:544:18at Object.module. anonymous FUNCTION as OPENSync ATOM_SHELL_ASAR.js:118:20
Object.fs.writeFileSyncfs.js:1155:15
Object.fsPlus.writeFileSync/opt/atom/resources/app.asar/node_modules/fs-plus/lib/fs-plus.js:240:17
File.module.exports.File.writeFileSync/opt/atom/resources/app.asar/node_modules/pathwatcher/lib/file.js:264:19
File.module.exports.File.writeFileWithPrivilegeEscalationSync/opt/atom/resources/app.asar/node_modules/pathwatcher/lib/file.js:362:21
File.module.exports.File.writeSync/opt/atom/resources/app.asar/node_modules/pathwatcher/lib/file.js:336:12at TextBuffer.module.exports.TextBuffer.saveAs/opt/atom/resources/app.asar/node_modules/text-buffer/lib/text-buffer.js:913:17
TextBuffer.module.exports.TextBuffer.save/opt/atom/resources/app.asar/node_modules/text-buffer/lib/text-buffer.js:899:19
TextEditor.module.exports.TextEditor.save/opt/atom/resources/app.asar/src/text-editor.js:554:26
Object.module.exports.autosavePaneItem/opt/atom/resources/app.asar/node_modules/autosave/lib/autosave.js:56:80
/opt/atom/resources/app.asar/node_modules/autosave/lib/autosave.js:27:26
```
### Commands ###
config.json
```cson
{
  core: {
    themes: atom - light - ui,
    monokai,
    disabledPackages: language - objective - c,
    language - c,
    language - clojure,
    language - csharp,
    language - java,
    language - toml,
    language - yaml,
    language - sql,
    language - mustache,
    language - php,
    quippet,
    followSymlinks: true,
    autoHideMenuBar: true
  },
  autosave: {
    enabled: true
  }
}
```
#### User 
```
autocomplete-plus, v2.12.0 
autocomplete-snippets, v1.2.0 
file-icons, v1.5.4 
linter, v0.12.0 
linter-shellcheck, v0.0.7 
minimap, v4.7.6 
monokai, v0.14.0
```

#### Dev
No dev packages

**Comment by 50Wliu** [comment]
Hi @Enkouyami did you by any chance change the formatting of this bug report?  It looks like all the code blocks and newlines were stripped out.

**Comment by Enkouyami** [comment]
Updated formatting.

**Comment by balexand** [comment]
I wasn't able to reproduce the exception, but I've verified that #35 makes it so broken symlinks will not be auto saved (`fs.isFileSync('broken-symlink.txt') === false`). I think this issue can be closed after #35 is merged.
